.travis.yml

```diff
@@ -9,24 +9,17 @@ services:
 
 env:
     # PHP 7.2
-    - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=nyholm SYMFONY_VERSION=3.4.* XDEBUG_VERSION=2.7.2
-    - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=nyholm SYMFONY_VERSION=4.2.* XDEBUG_VERSION=2.7.2
-    - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=nyholm SYMFONY_VERSION=4.3.* XDEBUG_VERSION=2.7.2
     - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=nyholm SYMFONY_VERSION=4.4.* XDEBUG_VERSION=2.7.2
-    - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=zendframework SYMFONY_VERSION=3.4.* XDEBUG_VERSION=2.7.2
-    - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=zendframework SYMFONY_VERSION=4.2.* XDEBUG_VERSION=2.7.2
-    - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=zendframework SYMFONY_VERSION=4.3.* XDEBUG_VERSION=2.7.2
+    - PHP_VERSION=7.2 PSR_HTTP_PROVIDER=nyholm SYMFONY_VERSION=5.0.* XDEBUG_VERSION=2.7.2
```

Comment:
After you rebase with master there shouldn't be any `XDEBUG_VERSION` here left and `SYMFONY_VERSION` became `SYMFONY_REQUIRE`.

Reply:
ok i think this is fixed after the merge from master.